### PR TITLE
fixes #199: show progress for install/uninstall

### LINKED ISSUE #199: Track progress for installations on experiment detail view.
We need to show another state to show the user that the experiment is installing.

### PR DESCRIPTION
**Pull request by meandavejustice**
fixes #199

### CODE REVIEW

**Comment by meandavejustice** [comment]
closed https://github.com/mozilla/idea-town/pull/319 in favor of this pr. Was having trouble cleaning up/squashing git history, so I just opened a clean branch. (This also fixes the commit message and branch name to reference the correct issue.)

File: addon/dev-prefs.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
   "BASE_URL": "http://ideatown.dev:8000",
-  "ALLOWED_ORIGINS": "http://ideatown.dev:8000/*, https://ideatown.firefox.com/*, http://idea-town-dev.elasticbeanstalk.com/*"
+  "HOSTNAME": "ideatown.dev",
+  "ALLOWED_ORIGINS": "ALLOWED_ORIGINS": "http://ideatown.dev:8000/*,https://ideatown.firefox.com/*,http://idea-town-dev.elasticbeanstalk.com/*"
```
Comment (lmorchard) [inline addon/dev-prefs.json]: Looks like an extra `"ALLOWED_ORIGINS":` copypasta'd here

Comment (meandavejustice) [inline addon/dev-prefs.json]: whoops!

**Comment by lmorchard** [comment]
So, on master branch, I can Enable and Disable back & forth many times in a row to get the addon installed/uninstalled. 

On this branch, there seems to be an issue with the addon installation state. I can enable & disable once. But, the next time I hit enable, it just returns back to the enable button state after flashing the progress animation. The addon is actually installed, but the button never changes state to "disable {test name}" until I reload the page. And then, if I disable and enable again, the disable button gets stuck in a similar way until I reload.

**Comment by meandavejustice** [comment]
@lmorchard Created issue for tracking installed/uninstalled state [here](https://github.com/mozilla/idea-town/issues/325)

**Comment by lmorchard** [comment]
Cool, going to merge this to get the good stuff in. We can circle back on #325 because hopefully repeated enable / disables are rare enough for now

**Pull request merged**